src/components/PersonComponent.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from 'classnames';
+import { PersonLink } from './PersonLink';
+import { Person } from '../types';
+import { useParams } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+const noParent = '-';
+
+export const PersonComponent = (props: Props) => {
+  const { person, people } = props;
+  const { slug, sex, born, died, fatherName, motherName } = person;
+  const { personID } = useParams();
+
+  const findPerson = (name: string): Person | undefined => {
+    return people.find(neededPerson => neededPerson.name === name);
+  };
```

Comment:
Prepare people with father/mother when you get the people array from the API.
`PersonComponent` is a presentational component, it should just accept props and render the UI